manifests/e/Electrum/Electrum/4.4.5/Electrum.Electrum.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Electrum.Electrum
+PackageVersion: 4.4.5
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine

```

Comment:
```suggestion
```
Because of Issue #94840, I suggest we remove this optional field from the manifest to unblock upgrades